src/modules/dpos/validator/utils/getMaxAmount.js

```diff
@@ -39,26 +49,36 @@ const getMaxAmount = async (wallet, network, voting, address) => {
         },
       }),
     },
-    nonce: wallet.sequence?.nonce,
     sender: {
-      publicKey: wallet.summary?.publicKey,
+      publicKey,
     },
     moduleCommand: MODULE_COMMANDS_NAME_MAP.voteDelegate,
   };
 
-  const maxAmountFee = await getTransactionFee({
-    token: 'LSK',
-    wallet,
-    network,
-    transaction,
-    selectedPriority: { title: 'Normal', value: 0, selectedIndex: 0 }, // Always set to LOW
-    numberOfSignatures: getNumberOfSignatures(wallet, transaction),
-  }, 'LSK');
+  const maxAmountFee = await getTransactionFee(
+    {
+      network,
+      transaction,
+      token: 'LSK',
```

Comment:
Should this be hard-coded? can we read it from the store?

Reply:
added todo